docs/documentation/Realtime/recent additions.md

```diff
@@ -0,0 +1,110 @@
+<a class="pencil-link" href="https://github.com/google/transit/edit/master/gtfs-realtime/CHANGES.md" title="Edit this page" target="_blank">
```

Comment:
We might want to remove the first two lines to remove the pencil icon.

Reply:
Link removed  - https://github.com/MobilityData/gtfs.org/pull/312/commits/87deccbbfc5943db330cebb92317c10de99da57c